Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs_description.md

```diff
@@ -0,0 +1,8 @@
+## BaseIntegration Help
+
+Markdown file for integration configuration  help snippet. In this file add:
+
+- Brief information about how to retrieve the API key of your product
+- Other useful information on how to configure your integration in XSOAR
+
+Since this is a Markdown file, we encourage you to use MD formatting for sections, sub-sections, lists, etc.
```

Comment:
Let's edit it - add some general info about the purpose of the integration.
Also, Add a section about how to generate the API-Key (can be either the github link or copy the steps from inside the link, it's up to you).